src/server/common_services/auth_server.js

```diff
@@ -139,14 +139,15 @@ function create_auth(req) {
                 system_id: system && system._id,
                 role: role_name,
                 extra: req.rpc_params.extra,
+                expiry: req.rpc_params.expiry,
```

Comment:
You have this declaration above at the beginning of the function:
```
    // var expiry = req.rpc_params.expiry;
```
Maybe remove it or use it?

Reply:
Will remove. Thanks